src/main/java/com/google/sps/model/TasksClient.java

```diff
@@ -38,6 +39,14 @@
    */
   List<TaskList> listTaskLists() throws IOException;
 
+  /**
+   * Get all task lists in a user's Tasks account with the specified task list titles
```

Comment:
Can you mention how the titles should be matched? I.e., exact match? title must be contained?

Reply:
Moved some methods into TasksClient in another PR and hence this is not needed anymore so I removed it. Thanks and apologies you had to review that.

Reply:
`RESOLVED`